.github/workflows/heroku_deploy.yml

```diff
@@ -0,0 +1,40 @@
+name: Dev deploy
+on:
+  # Runs on the push to the develop branch
+  push:
+    branches:
+      - develop
+  workflow_dispatch:
```

Comment:
The workflow_dispatch is empty. I think because we are not using it, its better to remove it. The same goes to the other .yml files

Reply:
The reason for the entry is to allow anyone to trigger the workflow on demand apart from the push and pull.

It is not empty just like that. It work same as on the push. When left empty it is just you don't have to provide entries apart from the data to be filled. 
The ability become accessible after merging to the main branch though.

Reply:
Okay